src/generator/contentful.clj

```diff
@@ -67,7 +69,12 @@
 (comment (get-contentful :asset-query {:$assetId "34YRcoaS4WJ5ORhpMlMHRM"}))
 (comment (get-contentful :entry-query {:entryId "782ka3lNsGXBrnE88Qf3jt"}))
 (comment (get-contentful :card-list-query {:listId "2pJ63nhY2QKbVesxgWOvq9"}))
-(comment (get-contentful :post-collection-query {:type ["news" "dev"]}))
+(comment 
+  
+  (get-contentful :post-collection-query {:type ["news" "dev" "article"]})
+  
+  )
```

Comment:
Code style empty rows, try to maintain context of your code by avoiding empty rows inside of a declaraction. Empty rows between two declaractions are alright but inside one it'll reduce readability.

In practice
```
; This is fine
(defn function
 []
 (println "Fn body")
 
(defn function2
 []
 (println "Fn body")
 
; this is not
(defn function
 []
 (println "fn first line")


(println "fn secondline))